cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -29,47 +29,33 @@ common project-config
                       -Wincomplete-uni-patterns
                       -Wpartial-fields
                       -Wredundant-constraints
-
-  if impl(ghc >= 8.10)
-    ghc-options:      -Wunused-packages
+                      -Wunused-packages
 
 library
   import:             base, project-config
-
-  -- We make use of features requiring ghc >= 8.10. In order to allow CI to
-  -- still build the 'all' component, simply make this non-buildable.
-  if impl(ghc < 8.10)
-    buildable: False
-  hs-source-dirs:      src
-  exposed-modules:
-    Test.Cardano.Ledger.TestableEra
-  build-depends:
-    shelley-spec-ledger
+  hs-source-dirs:     src
+  exposed-modules:    Test.Cardano.Ledger.TestableEra
+  build-depends:      shelley-spec-ledger
 
 benchmark bench
   import:             base, project-config
-
-  type:             exitcode-stdio-1.0
-  hs-source-dirs:
-    bench
-  main-is:          Main.hs
-  other-modules:
-    Bench.Cardano.Ledger.ApplyTx
-    Bench.Cardano.Ledger.Serialisation.Generators
-  build-depends:
-    bytestring,
-    cardano-binary,
-    cardano-ledger-shelley-ma-test,
-    cardano-ledger-shelley-ma,
-    containers,
-    criterion,
-    data-default-class,
-    deepseq,
-    QuickCheck,
-    shelley-spec-ledger,
-    shelley-spec-ledger-test
-  ghc-options:
-      -threaded
-      -rtsopts
-      -with-rtsopts=-N
-      -O2
+  type:               exitcode-stdio-1.0
```

Comment:
Is there a reason for this formatting change?

Reply:
I have rolled back the formatting changes and will squash on approval.